client/components/App.jsx

```diff
@@ -17,10 +17,12 @@ const App = () => {
   const [rating, setRating] = useState(0);
 
   useEffect(() => {
-    axios.get('/api/products')
+    const productId = window.location.search.split('=')[1];
+
+    axios.get(`/api/products/${productId}`)
       .then(({ data }) => {
-        setCurrentProduct(data[0]);
-        getStarRatingAvg(data[0].id)
+        setCurrentProduct(data);
+        getStarRatingAvg(data.id)
           .then((result) => setRating(result));
```

Comment:
i predict a merge conflict here because I refactored getStarRatingAvg to pass down total review count as well.

Reply:
I can look at that to check and modify for any breaking changes.